.github/workflows/smoke-test.yaml

```diff
@@ -40,24 +41,32 @@ jobs:
           restore-keys: |
             yarn-
 
-      - name: Install dependencies and Build Framework
+      - name: Install framework dependencies
         run: |
           yarn install --immutable
-          yarn build:clean || echo "Project already clean"
-          yarn build:js
 
       - name: Setup test project
         id: setup_test_project
         uses: ./.github/actions/setup_test_project
 
-      - name: Run `rw info`
+      - name: Install Playwright deps
+        run: npx playwright install --with-deps chromium
+
+      - name: Run `rw build`
         run: |
-          yarn rw info
+          yarn rw build
         working-directory: ${{ steps.setup_test_project.outputs.test_project_path }}
 
-      - name: Run `rw build` without prerender
+      # We run prisma reset, seed, rw dev, rw build and rw serve
+      - name: Run smoke tests
```

Comment:
```suggestion
      - name: Run smoke tests on 'rw dev' and 'rw serve'
```

Thinking out loud here — you did a really elegant job stitching this flow together. My (not well stated) intentions were to make this workflow as granular as possible and to use it for Telemetry data benchmarks (which is admittedly in tension with elegance and integration). 

So don't make any changes to the flow you've created, but I want to make sure that build, prisma, etc. are still handled as distinct checks outside both the `build:test-project` and `./tasks/smoke-test`. Thus the simple change in comment and title. But also just trying to better communicate my intent for  the overall purpose.